src/UsersReducer.js

```diff
@@ -5,11 +5,15 @@ import _ from 'lodash';
 import 'react-native-get-random-values';
 import {v4 as uuidv4} from 'uuid';
 
-const generateIds = (users: User[]): User[] => {
-  return users.map(u => ({
-    ...u,
+const generateIfNoId = (user: User) => {
```

Comment:
The function is confusing me. the name has `if` but the implementation has no `if`